entrypoint.sh

```diff
@@ -183,7 +183,8 @@ if [[ $COMMAND == 'plan' ]]; then
   # Meaning: 0 = Terraform plan succeeded with no changes. 2 = Terraform plan succeeded with changes.
   # Actions: Strip out the refresh section, ignore everything after the 72 dashes, format, colourise and build PR comment.
   if [[ $EXIT_CODE -eq 0 || $EXIT_CODE -eq 2 ]]; then
-    CLEAN_PLAN=$(echo "$INPUT" | sed -r '/^(An execution plan has been generated and is shown below.|No changes. Infrastructure is up-to-date.)$/,$!d') # Strip refresh section
+    CLEAN_PLAN=$(echo "$INPUT" | sed -r '/^(An execution plan has been generated and is shown below.|Terraform used the selected providers to generate the following execution
+plan.|No changes. Infrastructure is up-to-date.)$/,$!d') # Strip refresh section
```

Comment:
Looks like this is expected to be truncated: https://github.com/hashicorp/terraform/blob/v0.15.0/command/views/operation_test.go#L93
(Also just keeping this as a one-liner.)
```suggestion
    CLEAN_PLAN=$(echo "$INPUT" | sed -r '/^(An execution plan has been generated and is shown below.|Terraform used the selected providers to generate the following execution|No changes. Infrastructure is up-to-date.)$/,$!d') # Strip refresh section
```